lgc/builder/YCbCrConverter.cpp

```diff
@@ -43,14 +43,27 @@ using namespace llvm;
 Value *YCbCrConverter::wrappedSample(YCbCrWrappedSampleInfo &wrapInfo) {
   SmallVector<Value *, 4> coordsChroma;
   YCbCrSampleInfo *sampleInfo = wrapInfo.ycbcrInfo;
-  Value *chromaWidth = wrapInfo.chromaWidth;
-  Value *chromaHeight = wrapInfo.chromaHeight;
 
-  if (wrapInfo.subsampledX)
-    chromaWidth = m_builder->CreateFMul(wrapInfo.chromaWidth, ConstantFP::get(m_builder->getFloatTy(), 0.5f));
+  Value *chromaWidth = nullptr;
+  Value *chromaHeight = nullptr;
 
-  if (wrapInfo.subsampledY)
-    chromaHeight = m_builder->CreateFMul(wrapInfo.chromaHeight, ConstantFP::get(m_builder->getFloatTy(), 0.5f));
+  if ((m_metaData.word5.lumaDepth > 1) && (m_metaData.word1.planes > 1)) {
+    SqImgRsrcRegHandler proxySqRsrcRegHelper(m_builder, m_imgDescsChroma[1], m_gfxIp);
```

Comment:
It looks wrapInfo.imageDesc* is used in this function instead of m_imgDescsChroma, are they same?
If possible, please replace m_imgDescsChroma with wrapInfo.imageDesc. thanks!

Reply:
I'm sorry for misunderstanding comments, m_imgDescsChroma should be replaced with wrapInfo.imageDesc.